### PR TITLE
Migrate Dropdown docs

### PR DESCRIPTION
**Pull request by yamgent**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, please explain: Migration

Part of #274.

**Testing instruction:**
Use Netlify to ensure that the documentation is rendered correctly.

### CODE REVIEW

File: docs/userGuide/components/dropdown.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      </ul>
+    </dropdown>
+    <dropdown text="Disabled" type="warning" disabled>
+      <li><a href="#dropdown">Action</a></li>
+    </dropdown>
+    <dropdown type="info">
+      <button slot="before" type="button" class="btn btn-info">Segmented</button>
+      <li><a href="#dropdown">Action</a></li>
+    </dropdown>
+    <div class="btn-group btn-group-justified">
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Add `<div><br></div>` before this line to have a gap before the btn-group

File: docs/userGuide/components/dropdown.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  <button slot="before" type="button" class="btn btn-info">Segmented</button>
+  <li><a href="#dropdown">...</a></li>
+</dropdown>
+// In a button group
+<div class="btn-group btn-group-justified">
+  <a href="#dropdown" class="btn btn-default" role="button">Left</a>
+  <dropdown>...</dropdown>
+  <a href="#dropdown" class="btn btn-default" role="button">Right</a>
+</div>
+// With slots you can handle some elements as native bootstrap
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: We should standardise with HTML comments.

File: docs/userGuide/components/dropdown.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<dropdown text="Action" type="primary">
+  <li><a href="#dropdown">Action</a></li>
+  <li role="separator" class="divider"></li>
+  <li><a href="#dropdown">Separated link</a></li>
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Using 2 `<li>` here doesn't match the above example of 4 `<li>`

**Comment by damithc** [comment]
Do we need this component at all? I can't think of a use case in a documentation website.

File: docs/userGuide/components/dropdown.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  </ul>
+</dropdown>
+```
+</tip-box>
+<br>
+
+## Dropdown Options
+Name | Type | Default | Description 
+--- | --- | --- | ---
+class | `String` | `null` | Additional classes to change the style of the Dropdown.
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: > ... change the style of the Dropdown.

This is misleading, as users can't do something like `<dropdown class="btn-danger">`. Should clarify that `class` is used for either `<button>` or `<a>` only.

Comment (Chng-Zhi-Xuan) [reply docs/userGuide/components/dropdown.md]: Also, maybe link to [Bootstrap Buttons](https://getbootstrap.com/docs/4.0/components/buttons/) for the user to see the available classes they can use.

**Comment by yamgent** [comment]
> Do we need this component at all? I can't think of a use case in a documentation website.

@damithc The navbars are using the dropdown components.

**Comment by Chng-Zhi-Xuan** [comment]
You can nest a `dropdown` component in other components like `panel` or `tipbox` for Author flexibility too if they require it.

**Comment by damithc** [comment]
> @damithc The navbars are using the dropdown components.

> You can nest a dropdown component in other components like panel or tipbox for Author flexibility too, if they require it.

Got it. 👍

File: docs/userGuide/components/dropdown.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+```html
+<dropdown text="Action" type="primary">
+  <li><a href="#dropdown">Action</a></li>
+  <li><a href="#dropdown">Another action</a></li>
+  <li><a href="#dropdown">Something else here</a></li>
+  <li role="separator" class="divider"></li>
+  <li><a href="#dropdown">Separated link</a></li>
+</dropdown>
+<!-- For segmented dropdown, ignore text and add a "before" slot -->
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Add a line break before this line.

File: docs/userGuide/components/dropdown.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+<dropdown type="info">
+  <button slot="before" type="button" class="btn btn-info">Segmented</button>
+  <li><a href="#dropdown">...</a></li>
+</dropdown>
+<div><br></div>
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Putting this in the Markup is not necessary, it just helps to make the example look more presentable.

File: docs/userGuide/components/dropdown.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+<!-- In a button group -->
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Add a line break before this line.

File: docs/userGuide/components/dropdown.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+<div class="btn-group btn-group-justified">
+  <a href="#dropdown" class="btn btn-default" role="button">Left</a>
+  <dropdown>...</dropdown>
```
Comment (Chng-Zhi-Xuan) [inline docs/userGuide/components/dropdown.md]: Should replace `<dropdown>...</dropdown>` with the actual example below including the comment, starting from `<!-- With slots .... -->`

**Pull request merged**